webapp/components/post_view/commented_on_files_message_container/commented_on_files_message_container.jsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+
+import * as Utils from 'utils/utils.jsx';
+
+export default class CommentedOnFilesMessageContainer extends React.PureComponent {
```

Comment:
This could be renamed to just `CommentedOnFilesMessage`. I only named it with container since I planning to refactor it to a view and a container, and I didn't want to have to change the imports. 0/5 on if we do this now though